app/src/js/components/Providers/index.js

```diff
@@ -36,24 +33,22 @@ const Providers = ({
               showSidebar ? 'page__content--shortened' : 'page__content'
             }
           >
-            <Switch>
+            <Routes>
+            <Route index element={<Navigate to="all" replace />} />
               <Route
-                exact
-                path="/providers"
-                render={(props) => (
-                  <ProvidersOverview {...props} queryParams={filteredQueryParams} />
-                )}
+                path="all"
```

Comment:
Would you mind explaining how this works? We have a top-level Route with path=all and then a sub-Route with path=all. How do we get to this sub-route if the top-level Route component is set to "all"? Wouldn't that always match and prevent us from getting to this element? 

Very likely just my tenuous understanding of the new routing scheme.

Reply:
I answered a similiar somewhere else I believe but I would be glad to huddle to review the routing philosophy.